stdlib/FileWatching/src/pidfile.jl

```diff
@@ -38,6 +42,7 @@ Optional keyword arguments:
  - `refresh`: Keeps a lock from becoming stale by updating the mtime every interval of time that passes.
      By default, this is set to `stale_age/2`, which is the recommended value.
  - `wait`: If true, block until we get the lock, if false, raise error if lock fails.
+ - `wait_msg`: Provide a string for it to be `@info` logged when waiting for the lock
```

Comment:
Maybe this should accept a function to print however wanted, as showing a `@info` log might be too opinionated?